net-libs/xdp-tools/xdp-tools-1.5.8.ebuild

```diff
@@ -11,14 +11,17 @@ SRC_URI="https://github.com/xdp-project/${PN}/archive/refs/tags/v${PV}.tar.gz ->
 
 LICENSE="GPL-2 LGPL-2.1 BSD-2"
 SLOT="0"
-KEYWORDS="amd64 ~arm arm64 ~loong ~ppc ppc64 ~riscv x86"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
 IUSE="+tools"
 
+# tests require root privileges
+RESTRICT+=" test"
+
 DEPEND="
 	dev-libs/libbpf:=
 	net-libs/libpcap
-	virtual/zlib:=
 	virtual/libelf
+	virtual/zlib:=
 "
 RDEPEND="${DEPEND}"
 BDEPEND="
```

Comment:
btw, no need to look at it now of course, but have you looked at how hard bpf-toolchain would be here?